crowbar_framework/app/views/barclamp/neutron/_edit_attributes.html.haml

```diff
@@ -14,8 +14,11 @@
 
       = select_field :networking_plugin, :collection => :networking_plugins_for_neutron
 
-      #mode_container
-        = select_field :networking_mode, :collection => :networking_modes_for_neutron
+      #ml2_mechanism_drivers_container
+        = string_field %w(ml2_mechanism_drivers)
```

Comment:
> The thing is that the list must be sortable.

Is that really true for the mech_driver? I thought that would only be the case for the type drivers but ICBW. If it's not needed to be sortable we could make it a group of checkboxes, couldn't we?